lib/ui/add_book_screen/add_book_screen.dart

```diff
@@ -441,6 +442,7 @@ class _AddBookScreenState extends State<AddBookScreen> {
               clearStartDate: _clearStartDate,
               clearFinishDate: _clearFinishDate,
             ),
+            const BookReadingTimeField(defaultHeight: defaultFormHeight),
```

Comment:
Maybe we should show a checkbox or a question with quesion "Change reading time?" or "Set custom reading time?". It is good anyway if you don't want to play with the UI I can do this after this is merged.

Reply:
Do you mean hide reading time by default, show the field only if "Set custom reading time" box is checked?

Reply:
Yeah, I think the reads might not know that the reading time will be set from the start and finish date. So it would be nice to indicate that this will overwrite the time.

Reply:
For now, changed the field text to "Set custom reading time".